config/windows.config

```diff
@@ -21,6 +21,7 @@ if platform == Platform.WINDOWS:
     if target_arch == Architecture.X86_64:
         build = 'x86_64-w64-mingw32'
 else:
+    os.environ['WIX'] = os.path.join(prefix, 'lib', 'wix')
```

Comment:
Just to get sure: prefix here points to build-tools ? or is it the cerbero prefix (to dist/arch directory) ?

Reply:
I've tried it and it is from build-tools. Setting this:
os.environ['WIX'] = os.path.join(prefix, '..', '..', 'build-tools', 'lib', 'wix')
works, but if you have a variable pointing directly to build-tools folder it would be better than my patch ;)